functionalDB/chapter.md

```diff
@@ -108,27 +109,28 @@ Similar to entities, creating an attribute is done using the *make-attr* functio
      {:pre [(contains? #{:db/single :db/multiple} cardinality)]}
     (with-meta (Attr. name value -1 -1) {:type type :cardinality cardinality})))
 ````
+
 There are a couple of interesting patterns used in this constructor function: 
 
-* We use Clojure’s _Design by Contract_ [ADD REF HERE] pattern to validate that the cardinality parameter is a permissible value
-* We use Clojure’s destructuring mechanism to provide a default value of *:db/single* if one is not given
-* We use Clojure’s metadata capabilities to separate between an attribute's data (name, value and timestamps) and its metadata (type and cardinality). In Clojure, metadata handling is done using the *with-meta* (setting metadata) and *meta* (reading metadata) functions.
+* We use Clojure’s _Design by Contract_ [ADD REF HERE] pattern to validate that the cardinality parameter is a permissible value.

```

Comment:
We have a policy against referencing Wikipedia articles (since in effect you're saying, "you can look this up online", which is trivially true). We can leave this unreferenced -- readers can look it up if they need to.